test/plugin/test_in_http.rb

```diff
@@ -985,6 +994,12 @@ def test_if_content_type_is_initialized_properly
     assert_equal $test_in_http_connection_object_ids[0], $test_in_http_connection_object_ids[1]
   end
 
+  def get(path, params, header = {})
+    http = Net::HTTP.new("127.0.0.1", PORT)
+    req = Net::HTTP::Get.new(path, header)
```

Comment:
I think always returning `200` on all paths might not appropriate, it would be better that it's configurable.
On the other hand, it's not worth to introduce such complexity until we find actual inconvenience.